xmhf/src/xmhf-core/xmhf-runtime/xmhf-dmaprot/arch/x86/vmx/dmap-vmx-internal.c

```diff
@@ -285,7 +285,7 @@ static void delay_us(uint64_t us)
     while ( rdtsc64()-start < cycles ) ;
 }
 
-#define IOMMU_WAIT_OP(reg, cond, sts, msg_for_false_cond)     \
+#define IOMMU_WAIT_OP(drhd, reg, cond, sts, msg_for_false_cond)     \
```

Comment:
Consider make this a function, instead of macro. This may decrease code size and allow SL to fit into 64 KiB.